e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -84,7 +85,7 @@ context('When fee record correction feature flag is enabled', () => {
 
       createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT).check();
       createFeeRecordCorrectionRequestPage.reasonCheckbox(RECORD_CORRECTION_REASON.OTHER).check();
-      cy.keyboardInput(createFeeRecordCorrectionRequestPage.additionalInfoInput(), 'Some additional info.');
+      cy.keyboardInput(createFeeRecordCorrectionRequestPage.additionalInfoInput(), 'Some additional info & some more!');
```

Comment:
Updated so we can check these special characters aren't encoded as HTML entities later on in the journey on the "check the information" screen (e2e test additions for this screen in a different PR)

Reply:
This is the only use of the `govukCharacterCount` textarea component in the E2ETF work I believe so I don't think this would be a concern for this PR

There are some other usages in other parts of TFM UI (e.g. cases 'add a comment') that I've just tested and these have the same issue, but these are out of scope of this feature ticket and might fall under DTFS. I've raised this with Christian about opening a bug ticket that could use the helper implemented here and I'll follow up on this separate to this PR

Reply:
The above has been raised as a bug ticket by the DTFS team thanks - `DTFS2-7687`

Reply:
That's great thanks @sw-mattw we will take care of the rest of the `govukCharacterCount` components as long as the helper function is in `libs/common` implementation across DTFS should be swift. Thanks